samples/drivers/espi/prj_mec15xxevb_assy6853.conf

```diff
@@ -8,6 +8,6 @@ CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE=n
 # Sample code doesn't handle ACPI host communication
 CONFIG_ESPI_PERIPHERAL_HOST_IO=n
 # Test SAF flash portal read/erase/write on EVB
-CONFIG_ESPI_SAF=y
+#CONFIG_ESPI_SAF=y
```

Comment:
would be nice to add a comment here explaining why this was disabled.

Reply:
done